### PR TITLE
docs(ootb connectors): document inbound connector

### PR DESCRIPTION
**Pull request by igpetrov**
## What is the purpose of the change

Document inbound connector capabilities.

## Are there related marketing activities

Not at the moment. Marketing is being processed at a separate stream.

## When should this change go live?

_Does this change need to be released before a certain date or milestone? If so, when?_

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by christinaausley** [comment]
Happy to provide a technical review here as well when you're ready.

File: docs/components/connectors/out-of-the-box-connectors/http-webhook.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+## Prerequisites (Self-Managed Camunda 8 Platform)
+
+To use the **HTTP Webhook Connector**, you need to [download](https://github.com/camunda/connectors-bundle/tree/main/connectors/webhook-connector/element-templates) and [install](https://docs.camunda.io/docs/next/components/modeler/desktop-modeler/element-templates/configuring-templates/) the latest element template with your Modeler.
+You would also need to make sure there is a network connectivity between a caller and your Camunda Connectors Runtime configured to receive HTTP POST requests.
+
+## Create a HTTP Webhook Connector task
+
+1. Start building your BPMN diagram with a **Start Event** building block.
+2. Change its template to a HTTP Webhook.
+3. Feel in all required properties.
```
Comment (markfarkas-camunda) [inline docs/components/connectors/out-of-the-box-connectors/http-webhook.md:23]: Typo: 'Feel in' should be 'Fill in'

File: docs/components/connectors/use-connectors.md [hunk 2]
```diff
@@ -29,6 +29,36 @@ Fields in the properties panel marked with an equals sign inside a circle indica
...
+## Creating and Inbound Connector task
+
+:::note
+Inbound Connectors are currently supported only in [Camunda Platform 8 Self-Managed](../../self-managed/about-self-managed.md).
+To be able to use an Inbound Connector, you would require to [install](https://docs.camunda.io/docs/next/components/modeler/desktop-modeler/element-templates/configuring-templates/) a related element template (for example, [generic webhook](https://github.com/camunda/connectors-bundle/tree/main/connectors/webhook-connector/element-templates) or [GitHub webhook](https://github.com/camunda/connectors-bundle/tree/main/connectors/github/element-templates) first.
+:::
+
+1. Start building your BPMN diagram with a **Start Event** building block.
+2. Change its template to a Inbound Webhook of your choice (e.g., generic webhook or GitHub).
+3. Feel in all required properties.
```
Comment (markfarkas-camunda) [inline docs/components/connectors/use-connectors.md:41]: Typo: 'Feel in' should be 'Fill in'

File: docs/components/connectors/use-connectors.md [hunk 3]
```diff
@@ -29,6 +29,36 @@ Fields in the properties panel marked with an equals sign inside a circle indica
...
+4. Complete your BPMN diagram.
+5. Deploy it to your Camunda 8 instance.
+
+![inbound connector](img/use-inbound-connector-template.png)
+
+## Configuring an Inbound Connector
+
+In order to be able to deploy and use inbound webhook, you would need to feel in several fields.
```
Comment (markfarkas-camunda) [inline docs/components/connectors/use-connectors.md:49]: Typo: 'Feel in' should be 'Fill in'

**Review by markfarkas-camunda — CHANGES_REQUESTED** [review]
Looks good, there are just a few typo.

File: docs/components/connectors/out-of-the-box-connectors/github-webhook.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+1. In the **Webhook Configuration** section, set the **Webhook ID**. This value is used to build an URL endpoint of your webhook. For example, given the `Webhook ID` value is `myWebhookPath`, then the complete webhook URL endpoint will be `http(s)://<base URL>/inbound/myWebhookPath`.
+2. Set the **GitHub secret**. This is a shared secret key that has to be defined in both your BPMN and GitHub webhook configuration page. The value is used to calculate HMAC authentication signature.
+3. Configure **Activation Condition**. For example, given GitHub triggers a webhook endpoint with a new PR payload `{"action": "opened", "pull_request": ...}`, the **Activation Condition** value might look like as `=(request.body.action = "opened")`. Leave this field empty to trigger your webhook everytime.
+4. Configure **Variable Mapping**. For example, given GitHub triggers a webhook endpoint with a new PR payload `{"action": "opened", "pull_request": { "url: "https://...", ... }}` and you would like to extract a PR URL as a process variable `prUrl`. In that case, the **Variable Mapping** might look as `={prUrl: request.body.pull_request.url}`.
```
Comment (mbiciin) [inline docs/components/connectors/out-of-the-box-connectors/github-webhook.md:35]:
```suggestion
4. Configure **Variable Mapping**. For example, given that GitHub triggers a webhook endpoint with a new PR payload `{"action": "opened", "pull_request": { "url: "https://...", ... }}` and you would like to extract a PR URL as a process variable `prUrl`, the **Variable Mapping** might look as `={prUrl: request.body.pull_request.url}`.

```
or 
```suggestion
4. Configure **Variable Mapping**. For example, GitHub triggers a webhook endpoint with a new PR payload `{"action": "opened", "pull_request": { "url: "https://...", ... }}` and you would like to extract a PR URL as a process variable `prUrl`. In that case, the **Variable Mapping** might look as `={prUrl: request.body.pull_request.url}`.

**Review by mbiciin — CHANGES_REQUESTED** [review]
Looks good to me. :). I have few minor suggestions. I added them in the comments. I would also suggest changing the screenshots docs/components/connectors/img/use-inbound-connector-template-filled.png and docs/components/connectors/img/use-inbound-connector-template.png to be consistent with these changes https://github.com/camunda/web-modeler/pull/3498/files concerning the description for condition and variables.

File: docs/components/connectors/out-of-the-box-connectors/github-webhook.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+## Prerequisites (Self-Managed Camunda 8 Platform)
+
+To use the **GitHub Webhook Connector**, you need to [download](https://github.com/camunda/connectors-bundle/tree/main/connectors/github/element-templates) and [install](https://docs.camunda.io/docs/next/components/modeler/desktop-modeler/element-templates/configuring-templates/) the latest element template with your Modeler.
+
+## Create a GitHub Webhook Connector task
+
+1. Start building your BPMN diagram with a **Start Event** building block.
+2. Change its template to a GitHub Webhook.
+3. Feel in all required properties.
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/github-webhook.md:22]:
```suggestion
3. Fill in all required properties.
```

File: docs/components/connectors/out-of-the-box-connectors/http-webhook.md (hunk 1 above)
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/http-webhook.md:23]:
```suggestion
3. Fill in all required properties.
```

File: docs/components/connectors/use-connectors.md (hunk 2 above)
Comment (igpetrov) [inline docs/components/connectors/use-connectors.md:41]:
```suggestion
3. Fill in all required properties.
```

File: docs/components/connectors/use-connectors.md (hunk 3 above)
Comment (igpetrov) [inline docs/components/connectors/use-connectors.md:49]:
```suggestion
In order to be able to deploy and use inbound webhook, you would need to fill in several fields.
```

**Comment by berndruecker** [comment]
I would look at this now. @igpetrov Can I do some changes and push it to this branch? I basically ask to avoid merge conflicts as much as possible :-) 
As I can imagine I might do some bigger additions - I could also do it after this PR is merged if you prefer?

**Comment by igpetrov** [comment]
Hi @berndruecker,

Please make changes on this branch. There's nothing currently pending from my side.

**Comment by berndruecker** [comment]
Done

**Comment by igpetrov** [comment]
Thanks @berndruecker for taking care of it.

I just wanted to give https://github.com/camunda-cloud/identity/issues/1439 a try and make it working otherwise we should cover creating permissions in docs somewhere.

I am taking care of it here https://github.com/camunda/camunda-platform/issues/77. Probably should come up with a resolution tomorrow.

**Comment by igpetrov** [comment]
Good day, @christinaausley.

This one is ready for review & merge.

Thank you!

**Review by christinaausley — APPROVED** [review]
Technical review in latest commit for styling and grammatical adjustments -- let me know if you have any questions about these!

**Pull request merged**